GNSSLogger/app/src/main/java/com/google/android/apps/location/gps/gnsslogger/MeasurementProvider.java

```diff
@@ -62,7 +63,7 @@
         @Override
         public void onProviderEnabled(String provider) {
           if (mLogLocations) {
-            for (SensorFusionListener logger : mLoggers) {
+            for (MeasurementListener logger : mLoggers) {
```

Comment:
Likewise this can be `listener : mListeners`.

Reply:
Done!